R/doeResponseSurfaceMethodology.R

```diff
@@ -21,34 +21,57 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...){
   op2  <- length(options[["rsmResponseVariables"]])
   op3  <- length(options[["rsmBlocks"]])
 
-  if (options[["designType"]] == "cube" && options[["buildDesignInv"]])
-    .cubeDesign(jaspResults, options, dataset)
-  if (options[["designType"]] == "star" && options[["buildDesignInv"]])
-    .starDesign(jaspResults, options)
+  placeholder <- createJaspTable(title = gettext("Response Surface Methodology"))
+  jaspResults[["placeholder"]] <- placeholder
+  placeholder$dependOn(options = c("rsmVariables","rsmResponseVariables", "contour","coef","anova","res","pareto","resNorm",
+                                   "resFitted","buildDesignInv","desirability","contour"))
+  rsm <- list()
+  #Placeholder table when the user inputs something. If an analysis gets picked, remove the table
```

Comment:
So this is really not the way to do this...

Deleting a table is not supported, so this may break at any moment.


Can you make sure that whatever table you always output (there should be at least one!) is done in the same way as here: https://github.com/jasp-stats/jaspFrequencies/blob/d77e3d46c049278778391937d6ce54dc5640d005/R/binomialtest.R#L165-L218